### DISCUSSION TITLE
Reference to popular and last notes

### DISCUSSION THREAD

**Discussion by danymat** [srid/emanote#299] [Ideas]
Hello,

After exploring this ZK (https://www.alexeyshmalko.com/), I had the idea of having a way for emanote to show us popular and last notes. 

What do you think ?
I can think of two possible implementations:
1. Having a template that can be written in the starting page to display them at a required location.
2. Sidebar/links

**Comment by srid** [comment]
@danymat This requires some thinking. To begin with, where would you get the data for popularity / last-modified-date in the first place?

If this data was available in the frontmatter/yaml, we could add it to `Query.hs` (like we do for `tags`).
